src/Sulu/Bundle/WebsiteBundle/Controller/ExceptionController.php

```diff
@@ -14,17 +14,22 @@
 use Sulu\Bundle\WebsiteBundle\Resolver\ParameterResolverInterface;
 use Sulu\Component\Webspace\Analyzer\RequestAnalyzerInterface;
 use Symfony\Bundle\TwigBundle\Controller\ExceptionController as BaseExceptionController;
+use Symfony\Component\Debug\Exception\FlattenException;
```

Comment:
Actually only that is the critical change, isn't it? Or was there a reason for the other changes happening here?